packages/devtools_app/lib/src/screens/vm_developer/vm_function_display.dart

```diff
@@ -22,22 +22,25 @@ class VmFuncDisplay extends StatelessWidget {
   final FuncObject function;
 
   @override
-  Widget build(BuildContext context) => VmObjectDisplayBasicLayout(
-        object: function,
-        generalDataRows: vmObjectGeneralDataRows(function),
-        sideCardDataRows: _functionDetailRows(function),
-        sideCardTitle: 'Function Details',
-      );
+  Widget build(BuildContext context) {
+    return VmObjectDisplayBasicLayout(
+      object: function,
+      generalDataRows: vmObjectGeneralDataRows(function),
+      sideCardDataRows: _functionDetailRows(function),
+      sideCardTitle: 'Function Details',
+    );
+  }
 
   /// Returns a list of key-value pairs (map entries)
   /// containing detailed information of a VM Func object [function].
   List<MapEntry<String, Widget Function(BuildContext)>> _functionDetailRows(
     FuncObject function,
   ) {
     String? boolYesOrNo(bool? condition) {
-      if (condition == true) return 'Yes';
-      if (condition == false) return 'No';
-      return null;
+      if (condition == null) {
+        return null;
+      }
+      return condition == true ? 'Yes' : 'No';
```

Comment:
can you do `condition ? 'Yes' : 'No'` here?

Reply:
Done